backend/src/tasks/dnstwist.ts

```diff
@@ -11,13 +11,11 @@ async function runDNSTwist(domain: Domain) {
     'dnstwist',
     ['-r', '--tld', './worker/common_tlds.dict', '-f', 'json', domain.name],
     {
-      cwd: process.cwd(),
-      env: process.env,
       stdio: 'pipe',
       encoding: 'utf-8'
     }
   );
-  const savedOutput = String(child.stdout);
+  const savedOutput = child.stdout;
```

Comment:
Have you double-checked that this works if you run the scan from docker as well?